mart/attack/adversary_wrapper.py

```diff
@@ -16,7 +20,7 @@ class NormalizedAdversaryAdapter(torch.nn.Module):
     External adversaries commonly take input of NCWH-[0,1] and return input_adv in the same format.
     """
 
-    def __init__(self, adversary):
+    def __init__(self, adversary: Callback):
```

Comment:
I don't think this is the correct type. This should really be named `external_adversary` so as not to be confused with MART's implementation of `Adversary`. The only thing we assume about `external_adversary` is that it is `Callable`.